examples/standalone-with-org-policies/main.tf

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  key_rotation_period_seconds = "2592000s" #30 days
+  location                    = "us-east4"
+}
+
+module "secured_data_warehouse" {
+  source                           = "../.."
+  org_id                           = var.org_id
+  data_governance_project_id       = var.data_governance_project_id
+  confidential_data_project_id     = var.confidential_data_project_id
+  non_confidential_data_project_id = var.non_confidential_data_project_id
+  data_ingestion_project_id        = var.data_ingestion_project_id
+  sdx_project_number               = var.sdx_project_number
+  terraform_service_account        = var.terraform_service_account
+  access_context_manager_policy_id = var.access_context_manager_policy_id
+  bucket_name                      = "bucket_example"
+  dataset_id                       = "dataset_example"
```

Comment:
I think we just made all the examples use the same string name for each of these variables from a recent PR. Named to limit characters and to match the example's goal.